tests/common/GenerateCsv.spec.ts

```diff
@@ -0,0 +1,23 @@
+import Datastore from '../../src/common/Datastore';
+import GenerateCsv from '../../src/common/GenerateCsv';
+
+describe('GenerateCsv', () => {
```

Comment:
You need to init the database for this spec. Otherwise it won't work if run directly
https://github.com/tech-greedy/singularity/blob/main/tests/replication/DealReplicationService.spec.ts#L11